src/stores/media/index.ts

```diff
@@ -135,24 +135,31 @@ export const useMediaStore = defineStore('media', {
     fetchState(): FetchState {
       if (this._searchType === ALL_MEDIA) {
         /**
-         * For all_media, we return the error for the first media type that has an error.
+         * For all_media, we return 'All media fetching error' if all types have some kind of error.
          */
-        const findFirstError = () => {
-          for (const type of supportedMediaTypes) {
-            if (this.mediaFetchState[type].fetchingError) {
-              return this.mediaFetchState[type].fetchingError
-            }
-          }
-          return null
-        }
         const atLeastOne = (property: keyof FetchState) =>
           supportedMediaTypes.some(
             (type) => this.mediaFetchState[type][property]
           )
 
+        const allMediaError = () => {
```

Comment:
This needs improvement for sure! I don't know if we need a `fetchingError` for the All content view, and if so, what shape it should have. For the purposes of Load more, we need `fetchingError` to be null if at least one media type does not have an error.